sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuCSVScan.scala

```diff
@@ -325,7 +326,12 @@ class CSVPartitionReader(
       isFirstChunk: Boolean): Table = {
     val hasHeader = isFirstChunk && parsedOptions.headerFlag
     val csvOpts = buildCsvOptions(parsedOptions, readDataSchema, hasHeader)
-    Table.readCSV(cudfSchema, csvOpts.build, dataBuffer, 0, dataSize)
+    try {
+      Table.readCSV(cudfSchema, csvOpts.build, dataBuffer, 0, dataSize)
+    } catch {
+      case e: Exception =>
+        throw new IOException(s"Error when processing file [$partFile]: $e", e)
```

Comment:
Nit: Adding the exception message is redundant since it's already included as the caused exception.
```suggestion
        throw new IOException(s"Error when processing file [$partFile]", e)
```
Applies to the other places where the exception is included twice, in the message and as a cause.

Reply:
Thank you very much! I will update them.